renovate.json

```diff
@@ -97,6 +97,12 @@
         "before 7am every monday"
       ]
     },
+    {
+      "matchPackageNames": [
+        "alpine"
+      ],
+      "allowedVersions": "3.16"
```

Comment:
Let's hope that renovate only knows alpine versions with two-part version number, and that the php guys add dockerfiles for every single alpine version

Reply:
They for sure won't. For PHP-8.2 they dropped the build for alpine 3.15, so at one point they will also drop alpine 3.16.

Reply:
At one point. But until i hope we have a better solution.
(Like forcing the date format in one or another way, or replacing NBSP with spaces for the tests)